src/com/oltpbenchmark/api/Worker.java

```diff
@@ -51,12 +51,11 @@ public class Worker implements Runnable {
     private TransactionLatencyRecord latencies;
     private TransactionLatencyRecord failureLatencies;
     private WorkerTaskLatencyRecord workerTaskLatencyRecord;
-
     private final Statement currStatement;
 
     // Interval requests used by the monitor
     private final AtomicInteger intervalRequests = new AtomicInteger(0);
-
+    private final Connection ll_conn;
```

Comment:
ll_conn needs to be closed when the thread is going down.

Reply:
Handled in another PR